package.json

```diff
@@ -39,8 +39,9 @@
     "codecov:ci": "codecov",
     "link": "lerna link",
     "clean": "lerna clean --yes",
-    "test": "jest",
-    "test:ci": "jest --runInBand --collectCoverage",
+    "test": "TZ=UTC jest",
+    "test:watch": "TZ=UTC jest --watch",
+    "test:ci": "TZ=UTC jest --runInBand --collectCoverage",
```

Comment:
I had some tests failing of the date/time utilities. After some googling this seemed to be the easiest fix and it worked. I hope this doesn't cause any issues in CI.

Reply:
Is there a way we can set this in the jest config?

Reply:
I can look into that on Monday. I'm on PTO today, but just hopped on for a minute.

Reply:
@jordan-a-young  I'm looking at this now. I went master to master thinking I would make a different branch and PR for this. Problem with that is that the tests all work fine right now. This original PR was getting pushed at about 9 or 10 PM, which I think is what made the tests fail. The particular tests that were failing had to do with checking whether a two dates were x number of days apart, meaning that these tests work fine during work hours. My time zone is currently 4 hours off UTC, so the issue will only show up in the last four working hours of the day.

That's a long way round of saying, if you're alright with it, I'm gonna put this PR in without the timezone adjustments to jest, since they'll pass right now. But I'll check this issue again in the evening and put in the fixes as their own PR. Does that work for you?

Reply:
Here's what I'm looking at for a solution: https://newbedev.com/how-do-i-set-a-timezone-in-my-jest-config

Let me know if you have any thoughts on it.

Reply:
That sounds good to me. I was wondering why you were having problems, but he hadn't heard of it before. I think the linked solution looks good too